fosdem-bare-metal-in-the-cloud/source/index.rst

```diff
@@ -0,0 +1,65 @@
+
+
+Bare Metal In The Cloud: Isn’t it Ironic?
+=========================================
+
+*by Dmitry Tantsur and Ilya Etingof, Red Hat*
+
+In this talk
+------------
+
+* Why allocating bare metal machines
+* Ironic introduction, architecture and workflows
+* Future features
+
+Why bare metal allocation
+-------------------------
+
+* Raw computing power
+* Hard-to-virtualize hardware e.g. GPU
+* Perfect isolation e.g. tenant security
+
+What's ironic
+-------------
+
+* OpenStack bare metal provisioning service
+* + OpenStack Nova driver
+* + Deployment Agent
+
+What's inside ironic
+--------------------
+
+* Conductor Service - work flow engine
+* API Service
+* Drivers
+
+.. image:: conceptual_architecture.png
+   :align: center
+   :scale: 70%
+
+Machine deployment workflow
+---------------------------
+
+* Power ON
+* PXE boot deploy image
+* Write user image
+* Reboot
+
+.. image:: ironic-sequence-pxe-deploy.png
+   :align: center
+   :scale: 70%
+
+Latest developments
+-------------------
+
+* Ansible-based deployment
+* Redfish out-of-band introspection
+* Redfish BIOS configuration management
+
+Upcoming features
+-----------------
+
+* Deploy Templates
+* Federation Capabilities
+* SmartNIC Support
```

Comment:
Do you have any clue about SmartNICs? Because I don't :)

Reply:
My understanding is that it's like a BMC on top of a NIC. So that we could run a neutron agent  inside the NIC and make networking tricks from ironic. %-D

I do not have much knowledge on that, but may be if we just mention that as a cool new feature, our cluelessness would not be so apparent...

I can read more on that if you think it's worth it.

Reply:
Hmm, we can have the last slide with just listing of the features we do not understand and do not consider too important :)